src/_scss/layouts/default/header/nav/_dropdownMenu.scss

```diff
@@ -21,6 +21,19 @@
         font-weight: $font-semibold;
         font-size: rem(15);
         line-height: rem(19);
+        .covid-newbadge {
```

Comment:
I would import `components/_newBadge.scss` here instead to prevent code duplication. Also if we need to update the styling for all "New" badges in the future, it can be done in one place.